packages/cads-buttons/lib/component.button.scss

```diff
@@ -72,13 +80,26 @@ $btn-secondary-padding:                 $spacing--xs;
     &:hover {
       background: $btn-secondary-bgcolor-hover;
     }
+
+    svg {
```

Comment:
I think we could clean this up further by creating a placeholder class called `%btn-svg` with a mixin that would take the position, top and an if statement for the left and right margin output? Let me know if this isn't clear.